graylog2-web-interface/src/components/event-notifications/event-notification-types/HttpNotificationForm.jsx

```diff
@@ -132,6 +133,14 @@ class HttpNotificationForm extends React.Component {
                            onValidationChange={this.onValidationChange}
                            url={config.url}
                            autofocus={false} />
+        <Checkbox id="skip_tls_verification"
+                  name="skip_tls_verification"
+                  label="Skip TLS verification"
+                  onChange={this.handleChange}
+                  value="skip_tls_verification"
```

Comment:
We don't need a `value` attribute since we already have the `checked` attribute, unless the value of `skip_tls_verification` is not a boolean which is not the case here